measured-rails.gemspec

```diff
@@ -13,15 +13,16 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/Shopify/measured-rails"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.files         = `git ls-files -z`.split("\x0")
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "measured", Measured::Rails::VERSION
 
   spec.add_runtime_dependency "railties", ">= 4.2"
```

Comment:
Is the `railties` runtime dependency still required then?

Reply:
For now, yes. We define a:
```ruby
module Measured::Rails
  class Railtie < ::Rails::Railtie
```

We can do that conditionally I guess. But it would be a functionality change with more risk than benefit IMO.

I'm trying to remove the full dependency on all of rails. But basically any of the pieces of rails will still use railties.